tests/manifests/test-object.yaml

```diff
@@ -18,5 +18,17 @@ spec:
   preservePoolsOnDelete: false
   gateway:
     port: 80
-    # securePort: 443
     instances: 1
+  security:
+    kms:
+      connectionDetails:
+        KMS_PROVIDER: vault
+        VAULT_ADDR: https://vault.default.svc.cluster.local:8200
+        VAULT_BACKEND_PATH: rook/ver2
+        VAULT_BACKEND: "v2"
+        VAULT_SECRET_ENGINE: kv
+        VAULT_SKIP_VERIFY: "false"
```

Comment:
How did you get the signed certificate? Should this be true?

Reply:
Sorry I didn't understand your question, this certificate was validated by k8s right. So no need of skipping the vault check(and secondly Rook <-> vault communication is not required for the RGW. if `kvengine` version checks fail RGW assumes engine is v2 in the `CheckRGWKMS()`